### DISCUSSION TITLE
Why does "Use monospaced fonts" not work on my phone?

### DISCUSSION THREAD

**Discussion by RCMoonpie** [FossifyOrg/General-Discussion#602] [Q&A]
I started to create an issue because "Use monospaced fonts" does not appear to be working on my new phone `Motorola Edge+ 2023 on Android 14`, whereas it was working in SMT _Simple Notes Pro_ v6.17.0 on my old phone `moto x4 on Android 9`.

As I was entering info into the issue form it hit me to first try installing _Fossify Notes_ v1.0.0 on my old phone to see if it works there.  Lo and behold the notes **are** visible in a monospaced font.

So the issue appears to lie in one or more versions of Android (10 to 14) and/or in my new phone itself.  This realization prompted me to post here first instead of opening an issue.

**Does anyone know or have a guess as to why the text notes are not visible in a monospaced font on my `Motorola Edge+ 2023 on Android 14`?**

**Comment by Aga-C** [comment]
I don't have a Motorola phone, but have you checked if another apps, that should be displayed with monospaced font, have it? I found a topic on Motorola's forum, where someone has a similar problem: https://forums.lenovo.com/t5/motorola-edge-30/Monospace-font-after-update-T1RD33-116-33-12/m-p/5255806.

**Comment by naveensingh** [comment]
I have a Motorola phone that I use for development, DroidSansMono font is indeed there in the system: 

<img src="https://github.com/user-attachments/assets/edac89d8-ff89-4c12-bf98-90862cfe51b6" width=600/>

Maybe it has something to do with how Motorola's 'Personalize' app changes font? Monospace doesn't seem to work in WhatsApp either (surround with ``).

**Comment by charliefrance** [comment]
It doesn't work on my Galaxy A13 either

**Reply by naveensingh** [reply]
Does `monospace` appear like `monospace` in other apps like WhatsApp?

**Reply by charliefrance** [reply]
In WhatsApp it isn't monospace when I do the `code block`.

**Reply by naveensingh** [reply]
Then it's probably a system issue.

**Comment by naveensingh** [comment]
Closing because this sounds like a system issue. 

Please create a [bug report](https://github.com/FossifyOrg/Notes/issues/new?template=bug_report.yml) if `monospace` is working in other apps.
